source/MaterialXRender/Handlers/TinyEXRImageLoader.h

```diff
@@ -0,0 +1,46 @@
+#ifndef MATERIALX_TinyEXRImageLoader_H
+#define MATERIALX_TinyEXRImageLoader_H
+
+#include <MaterialXRender/Handlers/ImageHandler.h>
+
+namespace MaterialX
+{
+/// Shared pointer to an TinyEXRImageLoader
+using TinyEXRImageLoaderPtr = std::shared_ptr<class TinyEXRImageLoader>;
+
+/// @class @TinyEXRImageLoader
+/// Disk image loader wrapper using TinyEXR
+///
+class TinyEXRImageLoader : public ImageLoader
```

Comment:
The fact that TinyEXR is used seems like an implementation detail, so maybe the class name could be just ExrImageLoader.